src/components/Dashboard/widgets/Wallet/SendForm.tsx

```diff
@@ -96,7 +93,7 @@ const SendForm = ({ onSubmit }: SendFormProps): ReactElement => {
         onChange={e => setRecepient(e.target.value)}
         onKeyDown={() => ''}
         value={recepient}
-        placeholder={'Enter '.concat(currency, ' address')}
+        placeholder={'Enter '.concat(TICKER, ' address')}
```

Comment:
For formatting strings, probably better to use template literals everywhere since we're already using them elsewhere rather than mixing in `.concat`.

Reply:
I have nuked all the concats 🦾